test/functional/services/remote/browserdriver_api/browserdriver_api.js

```diff
@@ -1,21 +1,21 @@
 import { EventEmitter } from 'events';
 
-import { createLocalChromedriverApi } from './chromedriver_local_api';
-import { createRemoteChromedriverApi } from './chromedriver_remote_api';
+import { createLocalBrowserdriverApi } from './browserdriver_local_api';
+import { createRemoteBrowserdriverApi } from './browserdriver_remote_api';
```

Comment:
Can we camel-case here too? `createLocalBrowserDriverApi` and `createRemoteBrowserDriverApi`? Maybe rename the files too to `browser_driver_local_api` and `browser_driver_remote_api`.

Reply:
Done.